tests/core/full_node/stores/test_full_node_store.py

```diff
@@ -246,6 +246,7 @@ async def test_basic_store(
         # Test adding non genesis peak directly
```

Comment:
do you know what this comment means?

Reply:
no idea im assuming this was supposed to be a peak from another chain that is not a fork but we should fail in that case not just handle it...